pymser_tutorial.ipynb

```diff
@@ -455,16 +455,16 @@
       "\n",
       "                           Augmented Dickey-Fuller Test\n",
       "==============================================================================\n",
-      "Test statistic for observable: -3.9280150347827982\n",
-      "P-value for observable: 0.0018380773527202343\n",
+      "Test statistic for observable: -3.928015034782818\n",
+      "P-value for observable: 0.0018380773527200977\n",
       "The number of lags used: 46\n",
       "The number of observations used for the ADF regression: 35201\n",
       "Cutoff Metrics :\n",
       "  1%: -3.430536 | The data is stationary with 99 % confidence\n",
       "  5%: -2.861622 | The data is stationary with 95 % confidence\n",
       " 10%: -2.566814 | The data is stationary with 90 % confidence\n",
       "\n",
-      "pyMSER Execution time:               19.379 s\n"
+      "pyMSER Execution time:               2.731 s\n"
```

Comment:
After each instance of the `pyMSER Execution time` line, there is always a comment saying something like

> "**As you can see the results are the same, but the execution time drops from 7 minutes to 20 seconds.**"

These comments are all outdated now, given that the execution times changed considerably.

Reply:
Good catch. I'll fix it.